www/docs/en/dev/guide/overview/project-structure.md

```diff
@@ -0,0 +1,105 @@
+
+
+### `package.json`
+
+TODO 
+
+### `config.xml`
+
+Configures your application and allows you to customize the behavior of your project. See also [config.xml reference documentation][config.xml ref]
+
+### `www/`
+
+Contains the project's web artifacts, such as .html, .css and .js files. As a cordova application developer, most of your code and assets will go here. They will be copied on a `cordova prepare` to each platform's www directory. The www source directory is reproduced within each platform's subdirectory, appearing for example in `platforms/ios/www` or `platforms/android/assets/www`. Because the CLI constantly copies over files from the source www folder, you should only edit these files and not the ones located under the platforms subdirectories. If you use version control software, you should add this source www folder, along with the merges folder, to your version control system.
+
+### `node_modules/`
+
+TODO
+Dependencies inistalled via NPM. 
+Base for `platforms` and `plugins`
+
+### `platforms/`
+
+Contains all the source code and build scripts for the platforms that you add to your project.
+
+> **WARNING:** When using the CLI to build your application, you should not edit any files in the /platforms/ directory unless you know what you are doing, or if documentation specifies otherwise. The files in this directory are routinely overwritten when preparing applications for building, or when plugins are re-installed.
+
+### `plugins/`
+
+Any added plugins will be extracted or copied into this directory.
+
+### `hooks/`
+
+This directory may contains scripts used to customize cordova-cli commands. Any scripts you add to these directories will be executed before and after the commands corresponding to the directory name. Useful for integrating your own build systems or integrating with version control systems.
+
+Refer to [Hooks Guide] for more information.
+
+### `merges/`
+
+Platform-specific web assets (HTML, CSS and JavaScript files) are contained within appropriate subfolders in this directory. These are deployed during a `prepare` to the appropriate native directory.  Files placed under `merges/` will override matching files in the `www/` folder for the relevant platform. A quick example, assuming a project structure of:
+
+```
+merges/
+|-- ios/
+| -- app.js
+|-- android/
+| -- android.js
+www/
+-- app.js
+```
+
+After building the Android and iOS projects, the Android application will contain both `app.js` and `android.js`. However, the iOS application will only contain an `app.js`, and it will be the one from `merges/ios/app.js`, overriding the "common" `app.js` located inside `www/`.
+
+### Version control
+
+It is recommended not to check in `platforms/` and `plugins/` directories into version control as they are considered a build artifact. Your platforms and plugins will be saved in config.xml & package.json automatically. These platforms/plugins will be downloaded when `cordova prepare` is invoked.
```

Comment:
This should say "Your platforms and plugins will be saved in package.json automatically."

We no longer save them into config.xml as of Cordova 9.

Reply:
Yeah, exactly why I created this PR. This information being buried in the CLI referenc means it is ignored and I think most users never have seen it - but it could be so useful to understand what Cordova is actually doing and how it works.

I would love if you could add more comment on inaccuracies or missing information of this type here. Will try to work it all in.

Reply:
I applied changes so this can be re-reviewed.